### DISCUSSION TITLE
Nach reset mit dem Recoverytool lässt sich Freetz nicht mehr flashen

### DISCUSSION THREAD

**Discussion by lukyxd** [Freetz-NG/freetz-ng#459] [Allgemein] [answered]
Hi,
nach einen Reset mit dem Fritz image recovery tool lässt sich meine Fritzbox 7362 SL nicht mehr mit Freetz flashen.
Das ist dass, was mir immer ausgegeben wird.

![Bildschirmfoto vom 2022-02-07 21-30-08](https://user-images.githubusercontent.com/34316353/152866869-9379df21-4f8e-409d-bad8-b07f440656e4.png)
(Manchmal versucht er auch einen Ping zu machen mehrere male und kommt zum selben Ergebnis.)

**Comment by fda77** [comment, marked as answer]
Ich würde mal raten dass du mit dem "nas" FTP  verbunden bist, nicht mit dem FTP des bootloaders, da es sehr viele "." bei der Wartezeit sind. Im Forum bekommst du wahrscheinlich mehr Antworten

**Reply by lukyxd** [reply]
Hi 
dies würde ich ausschließen, da ich die Fritz Box auf Werkseinstellung gesetzt habe und dann versucht habe Freetz zu flashen. Damit gibt es ja das NAS FTP noch nicht, wenn ich es richtig verstanden habe.
Die Punkte kommen daher, da ich die Fritzbox erst noch am Strom hatte und die noch verbunden war mit meinen Rechner. Jetzt habe ich die mal vom Strom genommen bevor ich "make push-firmware" gestartet habe, dies hat aber leider nichts geholfen. Selbst den Rechner an LAN 1 zu stecken brachte leider nicht den gewünschten erfolg.

**Reply by lukyxd** [reply]
Danke an alle die mir geholfen haben. Ich habe es hinbekommen. Ich habe noch mal das Recovery tool angewand so wie in der Freetz Doku beschrieben danach ging es.

**Comment by fda77** [comment]
> Damit gibt es ja das NAS FTP noch nicht, wenn ich es richtig verstanden habe.

Ja sieht so aus. Die "connection refused" sprechen dafür. Ich hab mit dem avm-nas keine Erfahrung.

Am sichersten ist
 - Stromstecker raus
 - push firmware wartet auf die Box, zeigt die Punkte
 - Stromstecker rein
